src/library_syscall.js

```diff
@@ -1367,10 +1367,12 @@ var SyscallsLibrary = {
     err('warning: untested syscall');
 #endif
     path = SYSCALLS.getStr(path);
+    var allowEmpty = flags & {{{ cDefine('AT_EMPTY_PATH') }}};
+    flags = flags & (~{{{ cDefine('AT_EMPTY_PATH') }}});
```

Comment:
Looks like this line can go inside the `if ASSERTIONS`

Reply:
Also, I wonder if we should just pass all flags to calculateAt so that we con't need to repeat these lines for each syscall?

Perhaps that can be a followup to keep this change precise/narrow.

Reply:
yes follow up